.github/workflows/interop-tests.yml

```diff
@@ -0,0 +1,34 @@
+name: Interoperability tests with GnuTLS and NSS
+on: [pull_request, push]
+jobs:
+  test:
+    runs-on: ubuntu-22.04
+    container:
+      image: docker.io/nhorman/fedora38-interop:latest
+      options: --sysctl net.ipv6.conf.lo.disable_ipv6=0
+      volumes:
+        - /home/runner/work/openssl/openssl:/openssl
+    timeout-minutes: 90
+    strategy:
+      fail-fast: false
+      matrix:
+        COMPONENT: [openssl-upstream-gnutls, openssl-upstream-nss]
+    env:
+      COMPONENT: ${{ matrix.COMPONENT }}
+    # Code below inspired by https://fedoramagazine.org/github-actions-use-podman-to-run-fedora-linux/
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I don't think this is necessary. First of all, there is no such parameter `repo-token` https://github.com/actions/checkout#usage. but there is `token` and it has the current repo token as a default value.